readthedocs/rtd_tests/tests/test_unresolver.py

```diff
@@ -372,8 +374,32 @@ def test_unresolve_invalid_scheme(self):
             "fttp://pip.readthedocs.io/en/latest/",
             "fttps://pip.readthedocs.io/en/latest/",
             "ssh://pip.readthedocs.io/en/latest/",
+            "javascript://pip.readthedocs.io/en/latest/",
             "://pip.readthedocs.io/en/latest/",
         ]
         for url in invalid_urls:
             with pytest.raises(InvalidSchemeError):
                 unresolve(url)
+
+        with pytest.raises(InvalidSubdomainError):
+            unresolve("https:///pip.readthedocs.io/en/latest/")
```

Comment:
Why this raises `InvalidSubdomainError`?

Reply:
A tripe `///` is interpreted as a URL without a domain.

Reply:
```suggestion
        with pytest.raises(InvalidSubdomainError):
            # Triple slash is not valid
            unresolve("https:///pip.readthedocs.io/en/latest/")
```